app/models/miq_policy.rb

```diff
@@ -11,12 +11,18 @@ class MiqPolicy < ApplicationRecord
                                  PhysicalServer
                                  Vm).freeze
 
-  POLICY_TOWHAT_APPLIES_TO_CLASSES = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }.freeze
+  def self.policy_towhat_applies_to_classes(opts = nil)
+    policyToWhatAppliesToClasses = TOWHAT_APPLIES_TO_CLASSES.inject({}) { |hsh, key| hsh[key] = ui_lookup(:model => key); hsh }
```

Comment:
Was going to suggest not using camelcase as per Ruby style, but then realized that we can drop this temporary variable entirely.